chewy.gemspec

```diff
@@ -11,13 +11,14 @@ Gem::Specification.new do |spec|
   spec.description   = 'Chewy provides functionality for Elasticsearch index handling, documents import mappings and chainable query DSL'
   spec.homepage      = 'https://github.com/toptal/chewy'
   spec.license       = 'MIT'
+  spec.required_ruby_version = '~> 3.0'
 
   spec.files         = `git ls-files`.split($RS)
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'activesupport', '>= 5.2' # Remove with major version bump, 8.x
-  spec.add_dependency 'elasticsearch', '>= 7.12.0', '< 7.14.0'
+  spec.add_dependency 'activesupport', '>= 6.1'
+  spec.add_dependency 'elasticsearch', '>= 8.11', '< 9.0'
```

Comment:
Is the plan here to do a major version bump of Chewy as well? So the chewy 7.x series will continue to get any bugfixes for ES7 gems, and the chewy 8.x series will support the ES8.x gems?

Reply:
> Is the plan here to do a major version bump of Chewy as well?
Yes, since the major version of the Chewy, is equal to major version of the ES. 

> So the chewy 7.x series will continue to get any bugfixes for ES7 gems, and the chewy 8.x series will support the ES8.x gems?
This my plan, but since I have never done this before, cannot say for sure, how hard it will be to support both versions at the same time.

Reply:
Hopefully it shouldn't be too much work to keep the `7.x` branch up to date with changes in the Elasticsearch Ruby client. The last minor release is `7.17` and we're not planning to release any new minors or add any new features for the moment, just addressing bugs or security issues, so patch releases only.